lib/dependencies/LoaderDependency.js

```diff
@@ -5,9 +5,13 @@
 "use strict";
 const ModuleDependency = require("./ModuleDependency");
 
+/** @typedef {import("../NormalModule")} NormalModule */
+
 class LoaderDependency extends ModuleDependency {
 	constructor(request) {
 		super(request);
+		/** @type {NormalModule} */
+		this.module = undefined;
```

Comment:
Already defined in ModuleDependency. It's not safe to assume that this is a NormalModule. i. e. could be something else when using the DllReferencePlugin.